python/gtsam/tests/test_NonlinearOptimizer.py

```diff
@@ -83,6 +84,59 @@ def test_graduated_non_convexity(self):
         actual = GncLMOptimizer(self.fg, self.initial_values, gncParams).optimize()
         self.assertAlmostEqual(0, self.fg.error(actual))
 
+    def test_gnc_params(self):
+        base_params = LevenbergMarquardtParams()
+        # Test base params
+        for base_max_iters in (50, 100):
+            base_params.setMaxIterations(base_max_iters)
+            params = GncLMParams(base_params)
+            self.assertEqual(params.baseOptimizerParams.getMaxIterations(), base_max_iters)
+        # Test printing
+        str(params)
+        # Test each parameter
+        for loss_type in (GncLossType.TLS, GncLossType.GM):
```

Comment:
Maybe run the yapf formatter on this file? Also please add newlines between each group so that it is easier to read.

Reply:
Ran yapf and added some new lines. Let me know if it's not like you expected.